wperf-test/arg-parser-arg-utils-tests.cpp

```diff
@@ -62,26 +62,37 @@ namespace arg_parser_arg_utils_tests
         {
             std::wstring input = L"This is a line that exceeds the width.";
             size_t max_width = 15;
-            std::wstring expected = L"This is a line\nthat exceeds\nthe width.";
+            std::wstring expected = L"This is a line\nthat exceeds the\nwidth.";
```

Comment:
This test is showing your funtion is not working correctly?
```
that exceeds the
1234512345123456
```
There are 16 characters in second line, is that correct if `max_width` is `15`?